src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -1,4 +1,6 @@
 <?php
+/** @noinspection PhpIncompatibleReturnTypeInspection */
```

Comment:
I added these warnings to cut down on the clutter in IntelliJ. I don't think we are likely to miss any issues because of them, but I can remove them if preferred.